src/components/charts/SummaryHorizontalBarChart/SummaryHorizontalBarChart.css

```diff
@@ -5,30 +5,55 @@
   background-color: var(--g3-color__white);
   width: 100%;
   height: 100%;
-  overflow: hidden;
   box-sizing: border-box;
 }
 
 .summary-horizontal-bar-chart svg {
   overflow: visible;
+  line-height: 1em;
 }
 
-.summary-horizontal-bar-chart__legend {
-  width: 20%;
-  float: left;
+.summary-horizontal-bar-chart__item {
+  margin: 6px 0;
+  line-height: 15px;
+  position: relative;
 }
 
-.summary-horizontal-bar-chart__legend-item {
+.summary-horizontal-bar-chart__item-label {
   white-space: nowrap;
   overflow-x: hidden;
   text-overflow: ellipsis;
   font-size: 14px;
   font-weight: normal;
-  line-height: 1em;
   letter-spacing: .03rem;
   color: var(--g3-color__gray);
-  margin: 6px 0;
-  text-align: right;
+}
+
+.summary-horizontal-bar-chart__item-value {
+  font-size: 10px;
+  font-weight: 600;
+  letter-spacing: .02rem;
+  color: var(--g3-color__base-blue);
+  position: relative;
+  top: 1px;
+}
+
+.summary-horizontal-bar-chart__item-label,
+.summary-horizontal-bar-chart__item-value {
+  user-select: none;
+  -ms-user-select: none;
+  -webkit-user-select: none;
+}
+
+.summary-horizontal-bar-chart__item-block-wrapper {
+  display: flex;
+}
+
+.summary-horizontal-bar-chart__item-block {
+  height: 11px;
+  position: relative;
+  top: 2px;
+  margin-right: 10px;
 }
 
```

Comment:
similar to my comment for `SummaryPieChart.css`, can we also add a block for `.summary-horizontal-bar-chart__toggle` and set both `float` and `font-size` as well?